packages/block-library/src/navigation/index.php

```diff
@@ -980,6 +980,8 @@ function block_core_navigation_get_fallback_blocks() {
 	if ( class_exists( 'WP_Navigation_Fallback' ) ) {
 		$navigation_post = WP_Navigation_Fallback::get_fallback();
 	} else {
+		// Refactoring required: this class must be guarded as it is available only in Gutenberg, and not in Core.
+		// phpcs:ignore Gutenberg.CodeAnalysis.ForbiddenFunctionsAndClasses.ForbiddenClassUsage
```

Comment:
This is the only file that has the `phpcs:ignore comment` included that is going to sync to WP core. Will it work correctly without the rule defined? I simply don't know how PHPCS works so the answer will be simple.

Reply:
It will not generate any errors if the ignored rule is not defined.
However, I've moved these to the `phpcs.xml.dist` file in 255e3edc03a2c6cd8f8284b48eefa8c07d8e40d8 to avoid cluttering the code with `phpcs:ignore` statements. I hope that these files will be refactored soon, allowing for the removal of the `exclude-pattern` statements.